scaper/core.py

```diff
@@ -1698,38 +1698,42 @@ def _generate_audio(self, audio_path, ann, reverb=None,
 
         with _set_temp_logging_level(temp_logging_level):
 
-            # Array for storing all tmp files (one for every event)
-            tmpfiles = []
-            with _close_temp_files(tmpfiles):
-                isolated_events_audio_path = []
+            # List for storing all generated audio (one array for every event)
+            event_audio_arrays = []
```

Comment:
now that I'm reviewing this, I think `event_audio_list` might be a better name after all?

Reply:
Sounds good.